### ISSUE TITLE
Allow to handle commands written in game's chat with Dynamic Script

### ISSUE THREAD

**Issue by petrmenzel** [Kromster80/kam_remake#410]
Is there any way how to set some of text sended via chat in game as DynamicEventst?

**Comment by Kromster80** [comment]
No, sending text into chat from Dynamic script is not available and should not be. You can use that same script to display texts in-game.

**Comment by petrmenzel** [comment]
No, I mean something different. Example:
Write: "add 30 logs" Events
---> add 30 logs to player Action

**Comment by Kromster80** [comment]
Ah, so you are referring to taking commands from chat!
That would be interesting.
However that requires a lot of work, because of multiplayer and replays - all commands would need to be processed exactly the same and at exact times for all players and in replay.

**Comment by reyandme** [comment]
Similar feature was implemented as 'custom console commands' in my fork
